google-http-client/src/main/java/com/google/api/client/util/Base64.java

```diff
@@ -36,35 +36,41 @@
    * @param binaryData binary data to encode or {@code null} for {@code null} result
    * @return byte[] containing Base64 characters in their UTF-8 representation or {@code null} for
    *         {@code null} input
-   * @see org.apache.commons.codec.binary.Base64#encodeBase64(byte[])
    */
   public static byte[] encodeBase64(byte[] binaryData) {
-    return org.apache.commons.codec.binary.Base64.encodeBase64(binaryData);
+    if(binaryData == null){
+      return null;
+    }
+    return com.google.common.io.BaseEncoding.base64().encode(binaryData).getBytes();
```

Comment:
I don't think we need the fully qualified path any more (no more name conflicts)